dcos/marathon.py

```diff
@@ -758,7 +758,7 @@ def show_pod(self, pod_id):
         :rtype: dict
         """
 
-        path = self._marathon_id_path_join('v2/pods', pod_id)
+        path = self._marathon_id_path_join('v2/pods', pod_id) + '::status'
```

Comment:
I tried, but that method escapes colons, so it ends up as e.g. `/v2/pods/foo%3A%3Astatus`.